src/app/modules/shared-components/components/page-navigator/page-navigator.component.ts

```diff
@@ -19,11 +23,49 @@ export class PageNavigatorComponent {
   @Output() watch = new EventEmitter<void>();
   @Output() stopWatch = new EventEmitter<void>();
 
+  @ViewChild('op') private op?: OverlayPanel;
+
+  overlayPanelMarginTop = true;
+
   isCurrentGroupWatched$ = this.userSessionService.session$.pipe(
     filter(isNotNullOrUndefined),
     map(data => this.groupId !== undefined && data.watchedGroup?.id === this.groupId)
   );
 
-  constructor(private userSessionService: UserSessionService) {
+  state$ = this.userSessionService.watchedGroup$.pipe(
```

Comment:
So as long as you load a page navigator, which is a header which is displayed for both item and group ... you fetch the root activities of the currently-watched group ???  I am afraid that's completely wrong. 😕
When you need to do is when the user click on "startWatch" on a group page, then you start fetching the activity list of the **current group**. 

But anyway, it is a generic page navigator, there is no reason to have group or item stuff... so `@Input() groupId?: string;` makes no sense,  `isCurrentGroupWatch ed$ ...` as well. Shit, I missed them at a previous review. 😔 What was there before was an abstract "watch" which is displayed if the input "allowWatching" is true... which was allowing any element to have the watch button or not, without know for what kind of object it was for.